src/stratis_cli/_error_reporting.py

```diff
@@ -19,6 +19,20 @@
 
 import dbus
 
+from dbus_client_gen import DbusClientMissingInterfaceError
+from dbus_client_gen import DbusClientMissingPropertyError
+from dbus_client_gen import DbusClientMissingSearchPropertiesError
+from dbus_client_gen import DbusClientUnknownSearchPropertiesError
+
+_DBUS_INTERFACE_MSG = (
+    "The version of stratis you are running expects a different "
+    "D-Bus interface than the one stratisd provides. Most likely "
+    "you are running a version that requires a newer version of "
+    "stratisd than you are running.")
+
+_STRATIS_CLI_BUG_MSG = ("Most likely there is a bug in stratis, the program "
```

Comment:
The problem could be caused by a bug in dbus-client-gen, i.e., one of the libraries that stratis depends on. But it is virtually certain that bugs that would cause one of these errors have been eliminated entirely from dbus-client-gen. So that leaves a bug in stratis itself as the much more likely cause.